js/source/legacy/brapi/PedigreeViewer.js

```diff
@@ -14,11 +14,13 @@ function PedigreeViewer(server,auth,urlFunc){
         var locationSelector = null;
         
         urlFunc = urlFunc!=undefined?urlFunc:function(){return null};
-                
-        pdgv.newTree = function(stock_id,callback){
+             
+        pdgv.newTree = function(stock_id,callback){   
```

Comment:
If `number_ancestors`/`ancestor_depth` is added as an argument here, it can be controlled without modifying the source, in case someone wants deeper trees.